src/runtime_src/core/tools/common/ReportCu.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __ReportCu_h_
+#define __ReportCu_h_
+
+// Please keep external include file dependencies to a minimum
+#include "Report.h"
+
+class ReportCu : public Report {
+ public:
+  ReportCu() : Report("cu", "Cus present on the device", true /*deviceRequired*/) { /*empty*/ };

```

Comment:
Please don't use acronym's.
The report name shoulbd something more helpful, something like "compute_units" and the description should as well "Status of the compute units" or something like that.